src/SessionView.cpp

```diff
@@ -362,6 +368,16 @@ int SessionView::buttonAction(int x, int y, bool on, bool inCardRoutine) {
 		}
 	}
 
+	else if (x == scaleModeButtonX && y == scaleModeButtonY) {
```

Comment:
I was originally going to implement the menu on the Select encoder button, but I noticed [a few lines above](https://github.com/dctucker/DelugeFirmware/blob/bdd2a6333a6147b36a61b3c86d34e125fadd66b9/src/SessionView.cpp#L353-L367) already want to make use of this to cancel arming and launching during playback, and I didn't want to break that, so it's going on the Scale button until we figure out a better way.

Alternatively we could only show the song menu on Select encoder when playback is inactive?

Reply:
Hmm... sopping playback to menu dive isn't ideal, so my vote would be sticking with the scale button for the moment. It does pose an interesting question of how to make things consistent with the current UI, so maybe in future it'd be worth doing a rewrite with the song-mode menu on the select knob and the canceling launch feature can be a different button press/combo?

Reply:
Yeah, I think a bit of community input on how many folks get a lot of use out of the current "select knob cancels" would help make this decision clearer. Having slept on it, I'm now thinking of adding a "Song" submenu to either the global settings menu and/or the sound editor menu.

I kinda want to be able to send note on events while modifying the tuning, kinda like a guitar player would, ya know?

Reply:
Hard agree w/r/t note on messages while tuning. Personally I like the settings sub-menu idea, but maybe the forum/discord is a better place to get community-wide opinions?